doc/nrf/protocols/matter/getting_started/adding_bt_services.rst

```diff
@@ -0,0 +1,71 @@
+You can integrate a variety of Bluetooth® LE services to your Matter-enabled application.
+The Bluetooth LE services can run next to the Matter stack and the Bluetooth LE Arbiter class can advertise them in a specific prioritization order.
+For now, this solution is available for Matter over Thread.
+
+You can implement a Bluetooth LE service in one of the following ways:
+
+* Taking advantage of :ref:`Bluetooth services <lib_bluetooth_services>` and their related samples in the |NCS|.
+  You can use these implementations as templates and use their defines and helper functions as examples.
+* Using `Bluetooth SIG's Assigned Numbers documentation`_ with predefined characteristics to create your custom implementation.
+* Writing the custom Bluetooth LE service from scratch.
+
+This guide describes the first case and uses :ref:`nus_service_readme` as an example.
+
+.. _ug_matter_device_adding_bt_services_ble_arbiter:
+
+Advertising arbiter for Bluetooth LE services
+*********************************************
+
+The Bluetooth LE Arbiter class enables easier coexistence of application components that want to advertise their Bluetooth LE services, for example during :ref:`ug_matter_network_topologies_commissioning_stages_discovery`.
+
+When several application services are active, they can define a request to the Arbiter with the desired priority.
+If the service with the highest priority stops advertising, the Arbiter automatically selects the next service in the queue.
+If the service that requests advertising has a higher priority than the service that is running when the request is submitted, the advertising is restarted using parameters defined in the new request.
+
+In the Matter implementation in the |NCS|, the following services are configured by default for all Matter applications:
+
+* Commissioning for Matter (priority ``0``)
+* DFU over Bluetooth LE (priority ``uint8 max``)
+
+Lower number has a higher priority.
+If you implement a custom Bluetooth LE service, you need to prioritize it with the Bluetooth LE Arbiter.
+
+The class was created for the Zephyr platform in the Matter SDK.
+For more information about what the class offers, see the `Bluetooth LE Arbiter's header file`_.
+
+.. _ug_matter_device_adding_bt_services_adding:
+
+Adding support for a Bluetooth LE service
+*****************************************
+
+To add support for a Bluetooth LE service implemented in the |NCS|, complete the following steps:
+
+1. Check the documentation of the :ref:`Bluetooth service <lib_bluetooth_services>` for its characteristics and functions.
+#. Write the implementation of the `Bluetooth LE Arbiter's Request struct`_, including the ``Init`` method and the ``priority`` parameter.
+#. In the application source file, add a call to the `Bluetooth LE Arbiter's InsertRequest function`_, with the reference to the ``Request`` structure.
+   This adds the service to the Arbiter queue.
+   You can for example use the service's ``StartServer`` method.
```

Comment:
```suggestion
#. Add the application code that instantiates and fills the `Bluetooth LE Arbiter's Request struct`_, including its ``priority` field.
#. Add the application code that call the `Bluetooth LE Arbiter's InsertRequest function`_, with the reference to the ``Request`` structure, when the service needs to be advertised.
This adds the service to the Arbiter queue.
```
There's no such thing as implementation of the structure. Also, do not refer to `StartServer` and `Init` as they are not part of any common interface - they can only be mentioned in the section below that uses a specific sample.